sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -77,7 +78,7 @@ dependencies {
   compile library.java.junit
   compile library.java.netty_handler
   compile library.java.netty_tcnative_boringssl_static
-  compile library.java.proto_google_cloud_bigquery_storage_v1beta1
+  compile library.java.proto_google_cloud_bigquery_storage_v1beta2
```

Comment:
```
suztomo@suztomo:~/beam$ ./gradlew clean
Starting a Gradle Daemon (subsequent builds will be faster)
Configuration on demand is an incubating feature.

FAILURE: Build failed with an exception.

* Where:
Build file '/usr/local/google/home/suztomo/beam/sdks/java/io/google-cloud-platform/build.gradle' line: 81

* What went wrong:
A problem occurred evaluating project ':sdks:java:io:google-cloud-platform'.
> Cannot convert a null value to an object of type Dependency.
  The following types/formats are supported:
    - Instances of Dependency.
    - String or CharSequence values, for example 'org.gradle:gradle-core:1.0'.
    - Maps, for example [group: 'org.gradle', name: 'gradle-core', version: '1.0'].
    - FileCollections, for example files('some.jar', 'someOther.jar').
    - Projects, for example project(':some:project:path').
    - ClassPathNotation, for example gradleApi().
  ```

I don't see the key `proto_google_cloud_bigquery_storage_v1beta2` in BeamModulePlugin.groovy. Would you check a typo?

Reply:
Ah, good catch. It should be proto_google_cloud_bigquerybeta2_storage_v1